test/test-spawn.c

```diff
@@ -1751,6 +1751,14 @@ TEST_IMPL(spawn_inherit_streams) {
   ASSERT(uv_pipe_open(&pipe_stdout_child, fds_stdout[1]) == 0);
   ASSERT(uv_pipe_open(&pipe_stdin_parent, fds_stdin[1]) == 0);
   ASSERT(uv_pipe_open(&pipe_stdout_parent, fds_stdout[0]) == 0);
+  ASSERT(uv_is_readable((uv_stream_t*) &pipe_stdin_child));
+  ASSERT(!uv_is_writable((uv_stream_t*) &pipe_stdin_child));
```

Comment:
This test is failing on [FreeBSD](https://ci.nodejs.org/job/libuv-test-commit-freebsd/997/) and [SmartOS](https://ci.nodejs.org/job/libuv-test-commit-smartos/1025/) as on these systems `pipe()` returns a bidirectional data channel. See: https://www.freebsd.org/cgi/man.cgi?pipe(2) and https://illumos.org/man/2/pipe.
The failure is:
```
not ok 203 - spawn_inherit_streams
# exit code 134
# Output from process `spawn_inherit_streams`:
# Assertion failed in test/test-spawn.c on line 1755: !uv_is_writable((uv_stream_t*) &pipe_stdin_child)
```